clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -192,14 +214,39 @@
      * @return                          The new values which have been set as described in postProcessParsedConfig.
      */
     public static Map<String, Object> postProcessReconnectBackoffConfigs(AbstractConfig config,
-                                                    Map<String, Object> parsedValues) {
+                                                                         Map<String, Object> parsedValues) {
         HashMap<String, Object> rval = new HashMap<>();
         if ((!config.originals().containsKey(RECONNECT_BACKOFF_MAX_MS_CONFIG)) &&
                 config.originals().containsKey(RECONNECT_BACKOFF_MS_CONFIG)) {
-            log.debug("Disabling exponential reconnect backoff because {} is set, but {} is not.",
+            log.info("Disabling exponential reconnect backoff because {} is set, but {} is not.",
```

Comment:
Since we log `warn` when exponential `RETRY_BACKOFF_MS` and `SOCKET_CONNECTION_SETUP_TIMEOUT_MS` is disabled, I think we should at least log `info` here.